bin/opam.ml

```diff
@@ -215,7 +224,7 @@ let submit ~token ~dry_run ~yes ~opam_repo ~pkgs_to_submit local_repo
     | Some { owner; repo } -> Text.rewrite_github_refs ~user:owner ~repo changes
     | None -> changes
   in
-  parse_remote_repo remote_repo >>= fun { owner = fork_owner; _ } ->
+  get_fork_owner ~fork_owner ~remote_repo >>= fun fork_owner ->
```

Comment:
Probably better to just pass in `fork_owner` here.

Reply:
Before this point, `fork_owner` is an optional value that is probably `None` for most users. After this point, `fork_owner` is a string.
Inlining part of `get_fork_owner` here wouldn't be very nice to read. How do you suggest writing that ?

Reply:
An option could make a function along the lines `val fork_owner_if_unknown : fork_owner:string option -> _` that calls `parse_remote_repo` if `fork_owner` is unknown.

Reply:
This was exactly the intent behind `get_fork_owner`.